apps/docs/content/guides/ai/hybrid-search.mdx

```diff
@@ -0,0 +1,232 @@
+---
+id: 'ai-hybrid-search'
+title: 'Hybrid search'
+description: 'Combine keyword search with semantic search to get both direct and contextual results.'
+subtitle: 'Combine keyword search with semantic search.'
+sidebar_label: 'Hybrid search'
+---
+
+Hybrid search combines [full text search](/docs/guides/database/full-text-search) (searching by keyword) with [semantic search](/docs/guides/ai/concepts#what-are-embeddings) (searching by meaning) in order to identify results that are both directly and contextually relevant to the user's query.
+
+In Postgres, “search” just means filtering records in a table using a search query.
```

Comment:
@charislam how critical are these Vale errors? It seems like the issues it's picking up are actually better the other way around (it changes meaning subtly). Do you have any strong opinions on these suggestions?

Reply:
Errors will be blocking one day, use your judgment for warnings and suggestions. In this case, I actually agree with Vale:

- If you're providing a definition for `search` in Postgres, then the `just` serves no purpose
- If you're saying `just` to mean "in comparison to some other, more complex definition of search," it's clearer to make the explicit comparison

Reply:
But I'm going to remove the `just` error actually, there are other use cases for `just` so it can get annoying

Reply:
Got it. The point of the sentence was to help demystify what we actually me by "search" in the context of Postgres - mainly for developers who have less experience on the SQL side (search can be a bit of a black box on other platforms). If there's a concern with the word `just` I think I'll just remove the entire sentence.